get_cpu_process/get_cpu_process.py

```diff
@@ -0,0 +1,48 @@
+import psutil
+# https://thispointer.com/python-check-if-a-process-is-running-by-name-and-find-its-process-id-pid/
+
+
+def is_running(provided_process_name: str) -> bool:
+    """
+    Takes the name of a process and
+    returns True if it is running,
+    False if it isn't"""
+    for process in psutil.process_iter():
+        if provided_process_name.lower() == process.name().lower():
+            return True
+    return False
+
+
+def get_pid(provided_process_name: str) -> int:
+    """
+    Takes the name of a process and
+    returns the process id if it
+    is running"""
+    for process in psutil.process_iter():
+        if provided_process_name.lower() == process.name().lower():
+            return process.pid
+    return "Process not found"
+
+
+def get_process_run_time(provided_process_name: str) -> str:
+    """
+    Takes the name of a process and
+    returns the process runtime"""
+    from datetime import datetime
```

Comment:
It is better to import at the beginning of the file.
And for `datetime` I use `import datetime as DT` (`DT.datetime`, `DT.date`, `DT.timedelta`) - a handy module shorthand

Reply:
Done 👍